common/metrics/metric_defs.go

```diff
@@ -1470,10 +1470,10 @@ var (
 	ReplicationTasksRecvBacklog                    = NewDimensionlessHistogramDef("replication_tasks_recv_backlog")
 	ReplicationTasksApplied                        = NewCounterDef("replication_tasks_applied")
 	ReplicationTasksFailed                         = NewCounterDef("replication_tasks_failed")
-	ReplicationTasksLag                            = NewTimerDef("replication_tasks_lag")
+	ReplicationTasksLag                            = NewDimensionlessHistogramDef("replication_tasks_lag")
```

Comment:
Oh whoops, yeah that's a good catch. I think it could cause issues for people who overwrote the bucket boundaries originally. I'd at least like to put a comment in the release notes--making a v2 of this metric is probably overkill though.